packages/ts/react-signals/test/ValueSignal.spec.tsx

```diff
@@ -137,6 +137,32 @@ describe('@vaadin/hilla-react-signals', () => {
       expect(valueSignal.value).to.equal('baz');
     });
 
+    it('should accept a callback after replace', (done) => {
+      const valueSignal = new ValueSignal<string>('a', config);
+      subscribeToSignalViaEffect(valueSignal);
+      valueSignal.replace('a', 'b').result.then(done);
+      const [, , params] = client.call.firstCall.args;
+      simulateReceivedChange(subscription, {
+        id: (params!.event as { id: string }).id,
+        type: 'replace',
+        value: 'b',
+        accepted: true,
```

Comment:
Do we need to add couple of tests to cover the case that the `accepted = false` in which the then callback should not be called? (same for the test in `NumberSignal.spec.ts`)